{"name": "test_login", "status": "failed", "statusDetails": {"message": "AssertionError: assert 300 == 200\n  -300\n  +200", "trace": "def test_login():\n        result = Api.login()\n>       assert 300 == result.status_code\nE       assert 300 == 200\nE         -300\nE         +200\n\ntests_api\\tests\\test_login.py:8: AssertionError"}, "steps": [{"name": "Login", "status": "passed", "start": 1578581419539, "stop": 1578581419681}], "attachments": [{"name": "log", "source": "4c3f85bd-884d-476b-be38-b929dc21f8ae-attachment.txt", "type": "text/plain"}], "start": 1578581419539, "stop": 1578581419683, "uuid": "06574a6c-056d-4bb3-adb0-f4946898a2cf", "historyId": "e9007b3c9eb0c43cc15689ad4e592c75", "testCaseId": "8c4e19530360d0e2bf33133a3ee53763", "fullName": "tests_api.tests.test_login#test_login", "labels": [{"name": "parentSuite", "value": "tests_api.tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "LoshmanovNA-PC"}, {"name": "thread", "value": "3260-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests_api.tests.test_login"}]}